src/modules/person/domain/person.spec.ts

```diff
@@ -1,34 +1,18 @@
-import { createMock, DeepMocked } from '@golevelup/ts-jest';
 import { Test, TestingModule } from '@nestjs/testing';
 import { ConfigTestModule, DatabaseTestModule, MapperTestModule } from '../../../../test/utils/index.js';
 import { LoggerModule } from '../../../core/logging/logger.module.js';
-import { KeycloakUserService } from '../../keycloak-administration/index.js';
-import { UsernameGeneratorService } from './username-generator.service.js';
 import { faker } from '@faker-js/faker';
 import { Person } from './person.js';
-import { KeycloakClientError } from '../../../shared/error/index.js';
+import { DomainError } from '../../../shared/error/index.js';
 
 describe('Person', () => {
     let module: TestingModule;
-    let kcUserServiceMock: DeepMocked<KeycloakUserService>;
-    let usernameGeneratorService: DeepMocked<UsernameGeneratorService>;
 
     beforeAll(async () => {
         module = await Test.createTestingModule({
             imports: [MapperTestModule, ConfigTestModule, DatabaseTestModule.forRoot(), LoggerModule.register('Test')],
-            providers: [
-                {
-                    provide: KeycloakUserService,
-                    useValue: createMock<KeycloakUserService>(),
-                },
-                {
-                    provide: UsernameGeneratorService,
-                    useValue: createMock<UsernameGeneratorService>(),
-                },
-            ],
+            providers: [],
         }).compile();
-        kcUserServiceMock = module.get(KeycloakUserService);
-        usernameGeneratorService = module.get(UsernameGeneratorService);
     });
```

Comment:
Creating a test-module is no longer needed for this test, can be removed

Reply:
5c35da27797e5774702f8a071e0d682ed84dce7b